clientconn.go

```diff
@@ -1477,7 +1477,7 @@ func (ac *addrConn) startHealthCheck(ctx context.Context) {
 	}
 	// Start the health checking stream.
 	go func() {
-		err := ac.cc.dopts.healthCheckFunc(ctx, newStream, setConnectivityState, healthCheckConfig.ServiceName)
+		err := internal.HealthCheckFunc(ctx, newStream, setConnectivityState, healthCheckConfig.ServiceName)
```

Comment:
Could this use the local variable `healthCheckFunc` instead?

Reply:
Yes, changed to use the local var.